dataflow/manual/content.tex

```diff
@@ -1735,6 +1735,32 @@ \section{Example: Live Variable}
 \flow{LiveSimple}{.33}{1}{Simple sequential program to illustrate live variable.  Intermediate analysis results are shown.}
 
 
+\section{Example: Reaching Definitions}
+
```

Comment:
Please give a simple explanation about the concept of reaching definition.

Reply:
Okay!